src/components/IdScanner/IdScanner.tsx

```diff
@@ -124,10 +125,12 @@ export const IdScanner: FunctionComponent<IdScanner> = ({
   isScanningEnabled = true,
   hasLimitedInterestArea = true
 }) => {
+  const [platform] = useState<string>(Platform.OS);
   const [hasCameraPermission, setHasCameraPermission] = useState(false);
-  const interestAreaLayout = hasLimitedInterestArea
-    ? getInterestAreaDimensions(barCodeTypes)
-    : undefined;
+  const interestAreaLayout =
+    hasLimitedInterestArea && platform === "android"
+      ? getInterestAreaDimensions(barCodeTypes)
+      : undefined;
```

Comment:
hmm not sure if this is the issue here, but currently im unable to successfully scan IDs or QRs. like the scanner no longer working

Reply:
also to clarify! so ios will not have any labels/guidance right? i.e. same as previously!

Reply:
The issue was actually another check below, fixed it!

For now, the decision is for iOS devices to not have any guidance. However, moving forward we should have labels for both iOS and Android. We're pending advice from our UX designers for the labels!

Reply:
okay thanks for clarifying! for the labels, will be included in this PR?

Reply:
Yes, the labels will be included in this PR!